presto-docs/src/main/sphinx/conf.py

```diff
@@ -94,6 +94,6 @@ def get_version():
 
 html_title = '%s %s Documentation' % (project, release)
 
-html_add_permalinks = ''
+html_add_permalinks = u'\u00A7'
```

Comment:
although both junit & jdbi use `§` for this, i kind of would expect `#` 

anyway, they both show it on the left side of text (here it's on the right)

Reply:
Sounds good to me